src/components/Cards/CardParticipante.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import imgAvatarDefault from "../../assets/icons/teste-avatar.svg"
+import "../../assets/css/Cards/Cardparticipante.css"
+import {
+  Card, CardImg, CardBody, CardDeck,
+  CardTitle, CardSubtitle
+} from 'reactstrap';
+
+/**
+ * Componente de avatar de participante
+ * 
+ * @param {object} props //nome (do participante) e img (avatar) 
+ */
+export const AvatarParticipante = (props) => {
+    let nome = props.nome; 
+    let img = props.avatar ? 
+        props.avatar :
+        imgAvatarDefault;
+
+    return (
+            <span id="card_avatar">
+                <CardImg src={img} alt={nome} />
+            </span>
+    );
+};
+
+/**
+ * Componente de identidade de participante
+ * (nome, cargo e link do CV)
+ * 
+ * @param {object} props //nome, cargo e linkedin
+ */
+export const IdentidadeParticipante = (props) => {
```

Comment:
mover IdentidadeParticipante para um outro arquivo e extrair props logo no início da funçao

Reply:
Feito !